### PR TITLE
Show event on Map #213

### PR DESCRIPTION
**Pull request by TigreBois**

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sdp/musiconnect/events/MyEventPageTests.java [hunk 1]
```diff
@@ -133,4 +134,21 @@ public void testDeleteYesButtonClick() {
...
+        eventPageRule.launchActivity(intent);
+
+        waitSeconds(3);
+
+        onView(withId(R.id.toMap)).perform(click());
+        onView(withId(android.R.id.button1)).perform(click());
+
+        waitSeconds(3);
+
+        intended(hasComponent(MapsActivity.class.getName()));
```
Comment (antoniopisanell) [inline app/src/androidTest/java/ch/epfl/sdp/musiconnect/events/MyEventPageTests.java:152]: Good test

Comment (TigreBois) [reply app/src/androidTest/java/ch/epfl/sdp/musiconnect/events/MyEventPageTests.java:152]: An event can't be created without a location anymore, can it?

Comment (antoniopisanell) [reply app/src/androidTest/java/ch/epfl/sdp/musiconnect/events/MyEventPageTests.java:152]: Yes the geocoder can fail and return null, or we can set a location as « at the studio »

Comment (TigreBois) [reply app/src/androidTest/java/ch/epfl/sdp/musiconnect/events/MyEventPageTests.java:152]: I though if the geocoder failed, it wouldn't upload the event; doesn't make much sense to have an event without a valid location

**Pull request merged**